bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostCB.scala

```diff
@@ -161,6 +166,42 @@ final class GhostCB(obsComp: ISPObsComponent) extends AbstractObsComponentCB(obs
             case _ =>
               // The asterism may not have been configured by this point.
         }}
+
+      val mags = GhostCB.magnitudes(getObsComponent().getContextObservation())
+      GhostCB.MagProperties.foreach { case (band, prop) =>
+        mags.get(band).foreach { value =>
+          config.putParameter(systemName, DefaultParameter.getInstance(prop, value))
+        }
+      }
     }
   }
+}
+
+object GhostCB {
+
+  val MagProperties: Map[MagnitudeBand, String] = Map(
+    MagnitudeBand._g -> Ghost.MAG_G_PROP,
+    MagnitudeBand.V  -> Ghost.MAG_V_PROP
+  )
```

Comment:
@cquiroz  Are these the right bands?

Reply:
Is that SloanG, I think we wanted Gaia G but we don't have that on the OT yet

Reply:
Will these be stored in the map under a single key?

Reply:
Yes, Sloan G.  Is this task still useful if we don't have Gaia G?  These are science targets not guide stars in any case.

Reply:
My understanding is that we'll add Gaia G in the near future, I'll confirm if Sloan G is useful

Reply:
From user feedback::
`The SloanG is a passable alternative to Gaia G, but V would be preferred.`

I think we should do V or else Sloan G and revesit when we have Gaia G

Reply:
Sounds good, both will be added if both are known.

Reply:
Should we go ahead with this then?  If we later add a new band it will be easy to swap in here.

Reply:
Yes, let's go ahead